packages/calypso-analytics/src/utils/is-country-in-gdpr-zone.ts

```diff
@@ -41,9 +41,9 @@ const GDPR_COUNTRIES = [
  * @returns Whether the country is in the GDPR zone
  */
 export default function isCountryInGdprZone( countryCode: string | undefined ): boolean {
-	if ( 'unknown' === countryCode ) {
+	if ( 'unknown' === countryCode || undefined === countryCode ) {
```

Comment:
`undefined` is another way how `countryCode` can be "unknown", let's respect it.